js/api/Geofencing.js

```diff
@@ -39,12 +37,12 @@ export function clearActiveMap(component) {
  */
 export function setupGeofences(callback) {
```

Comment:
The bindings to the background geolocation library are missing in the iOS project, how is this code to function on iOS?